src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,150 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    public static final String NAME1 = "oleg";
+    public static final String NAME2 = "stepan";
+    public static final String NAME3 = "oleksiy";
+    public static final String PASS1_VALID = "123456";
+    public static final String PASS2_VALID = "6543210";
+    public static final String PASS3_VALID = "123456123";
+    public static final int AGE1_VALID = 28;
+    public static final int AGE2_VALID = 22;
+    public static final int AGE3_VALID = 40;
```

Comment:
It’s bad practice to use counters for names.  you can name your variables like ```OLEG_NAME```, ```VALID_PASSWORDS_SIX_SYMBOLS``` etc.